pandas/io/parsers.py

```diff
@@ -183,41 +200,34 @@ class ParserWarning(Warning):
 warn_bad_lines : boolean, default True
     If error_bad_lines is False, and warn_bad_lines is True, a warning for each
     "bad line" will be output. (Only valid with C parser).
-infer_datetime_format : boolean, default False
-    If True and parse_dates is enabled for a column, attempt to infer
-    the datetime format to speed up the processing
-skip_blank_lines : boolean, default True
-    If True, skip over blank lines rather than interpreting as NaN values
 
 Returns
 -------
 result : DataFrame or TextParser
 """
 
-_csv_params = """sep : string, default ','
-    Delimiter to use. If sep is None, will try to automatically determine
-    this. Regular expressions are accepted.
-engine : {'c', 'python'}
+# engine is not used in read_fwf() so is factored out of the shared docstring
+_engine_doc = """engine : {'c', 'python'}, optional
     Parser engine to use. The C engine is faster while the python engine is
     currently more feature-complete."""
 
-_table_params = """sep : string, default \\t (tab-stop)
-    Delimiter to use. Regular expressions are accepted.
-engine : {'c', 'python'}
-    Parser engine to use. The C engine is faster while the python engine is
-    currently more feature-complete."""
+_sep_doc = """sep : str, default {default}
+    Delimiter to use. If sep is None, will try to automatically determine
+    this. Regular expressions are accepted and will force use of the python
+    parsing engine and will ignore quotes in the data."""
 
 _read_csv_doc = """
 Read CSV (comma-separated) file into DataFrame
 
 %s
-""" % (_parser_params % _csv_params)
+""" % (_parser_params % (_sep_doc.format(default="','"), _engine_doc))
 
 _read_table_doc = """
 Read general delimited file into DataFrame
 
 %s
-""" % (_parser_params % _table_params)
+""" % (_parser_params % (_sep_doc.format(default="\\t (tab-stop)"),
```

Comment:
This doesn't render as `'\t'` in the [html](http://pandas.pydata.org/pandas-docs/stable/generated/pandas.read_table.html) (current behavior), but `\\\\t` would, however then the unrendered docstring looks a little strange so I left it as is.